compiler/rustc_ty_utils/src/abi.rs

```diff
@@ -834,11 +710,6 @@ fn fn_abi_adjust_for_abi<'tcx>(
                     }
                 }
```

Comment:
Maybe move this code into adjust_for_rust_abi too?

Reply:
This was my first attempt locally. The function `fn_abi_adjust_for_abi` uses items from the `rustc_middle` and `rustc_session` crates, but `rustc_target` crate doesn't seem to depend on them. I guess this is to keep `rustc_target` crate as minimal as possible. So I gave up on moving it all over.